### DISCUSSION TITLE
os error 111

### DISCUSSION THREAD

**Discussion by satyam455** [Cyfrin/foundry-full-course-cu#48] [Q&A]
Error: 
error sending request for url (http://localhost:8545/): error trying to connect: tcp connect error: Connection refused (os error 111)

Context:
- Error #0: error trying to connect: tcp connect error: Connection refused (os error 111)
- Error #1: tcp connect error: Connection refused (os error 111)
- Error #2: Connection refused (os error 111)

i'm running in anvil!!

**Comment by satyam455** [comment]
solved

**Reply by cromewar** [reply]
Hey @satyam455 can you please share what was the problem and how you solved it so it might be useful for the other students?

**Reply by starfish1905** [reply]
how?

**Reply by satyam455** [reply]
Hey it's been long time I have written this error I forgot what was that sorry.

**Comment by Equious** [comment]
Closing issue due to lack of response.

**Comment by starfish1905** [comment]
how?
